static/js/settings_org.js

```diff
@@ -437,23 +527,27 @@ function _set_up() {
                 if (response_data.allow_message_editing !== undefined) {
                     // We expect message_content_edit_limit_seconds was sent in the
                     // response as well
-                    var data_message_content_edit_limit_minutes =
-                        Math.ceil(response_data.message_content_edit_limit_seconds / 60);
+                    var unit_time_object =
+                        conversion_form_seconds(response_data.message_content_edit_limit_seconds);
+
+                    // updates the new edit limit in change_content_edit_limit button in
+                    // "organization settings" and also it sets "No limit" for 0 min.
+                    exports.update_message_edit_time_limit_settings(unit_time_object);
+
                     if (response_data.allow_message_editing) {
                         if (response_data.message_content_edit_limit_seconds > 0) {
                             ui_report.success(
-                                i18n.t("Users can now edit topics for all their messages, and the content of messages which are less than __num_minutes__ minutes old.",
-                                       {num_minutes : data_message_content_edit_limit_minutes}),
-                                message_editing_status);
+                                i18n.t("Users can now edit topics for all their messages, and the content of messages which are less than __num_time__ __unit__ old.",
```

Comment:
This is going to be complicated to translate. With a modal I think we need no success message at all, since it's visibly clear that the setting was changed, so let's just remove this.

Reply:
So this means `admin-realm-message-editing-status` is of no use further as all changes are visible on button.